### PR TITLE
Add back Robocopy cleanup 

### PR DESCRIPTION
**Pull request by MattGal**
but attempt to kill it before the timeout elapses to prevent leaked processes.

### CODE REVIEW

**Comment by MattGal** [comment]
@chcosta any feedback?

File: src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs [hunk 1]
```diff
@@ -29,11 +29,14 @@ public class CleanupVSTSAgent : Microsoft.Build.Utilities.Task
 
         public int MaximumWorkspacesToClean { get; set; } = 8;
 
+        public bool AttemptMaxPathFolderRemoval { get; set; } = true;
```
Comment (chcosta) [inline src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs]: nit: `AttemptLongPathRemoval` this feels slightly more accurate

File: src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs [hunk 2]
```diff
@@ -29,11 +29,14 @@ public class CleanupVSTSAgent : Microsoft.Build.Utilities.Task
...
+
         public int? SleepTimeInMilliseconds { get; set; }
         public ITaskItem[] ProcessNamesToKill { get; set; }
 
         private static readonly int s_DefaultRetries = 3;
         private static readonly int s_DefaultSleepTime = 2000;
+        private DateTime timerStarted;
```
Comment (chcosta) [inline src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs]: according to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md, this should be `_timeStarted`

File: src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs (hunk 1 above)
Comment (MattGal) [inline src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs]: I don't mind changing it but I still maintain this is only going to be an attempt :)

**Comment by MattGal** [comment]
Suggested changes made, will wait til CI build finishes and merge.

**Pull request merged**